neuronpp/utils/record.py

```diff
@@ -143,17 +146,42 @@ def _plot_animate(self, steps=10000, y_lim=None, position=None):
 
                 ax.set_xlim(t.min(), t.max())
                 if y_lim is None:
-                    ax.set_ylim(r.min()-(np.abs(r.min()*0.05)), r.max()+(np.abs(r.max()*0.05)))
+                    ax.set_ylim(r.min() - (np.abs(r.min() * 0.05)), r.max() + (np.abs(r.max() * 0.05)))
 
                 # update data
                 line.set_data(t, r)
-
+                # info draw triangles for true and predicted classes
+                true_x, true_y, pred_x, pred_y = self._class_tcks(label=i, true_class=true_class, pred_class=pred_class,
```

Comment:
i is not a label. It is an enumerator of sections to records with the same variable name (eg. v)

Reply:
Yes, this is an error here, or rather it is used in assumption that labels are 0, 1, 2, ... They need not to be. I shall pass a table of class_names as parameter, consistent with enumeration, and then see line in line 177 we shall have
if true_class[k] == class_names[label], and similarly in 179: if pred_class == true_classes[label]

Reply:
`i` represents number of segment saved to be recorded. If you want to record from 5 neurons - 2 input, 3 output in the same `Record` object - you can. So in this case `i` won't represent any true values, but rather enumerator. 

I think you should add parameter to the `Record` object to show predicted/true values, but it must be done explicitely as optional param: in the constructor or in the plotting method. Because otherwise it is extremelly error prone.